cvmfs/util.cc

```diff
@@ -927,15 +906,6 @@ bool HasSuffix(
 }
 
 
-bool IsNumeric(const std::string &str) {
```

Comment:
In a currently pending [Pull Request](https://github.com/cvmfs/cvmfs/pull/885) I am actually [using this function](https://github.com/cvmfs/cvmfs/pull/885/files#diff-e97df2d1b1380e814faaf512cc866e46R142). Please don't remove it.